ma_statement.c

```diff
@@ -1795,13 +1793,15 @@ SQLRETURN MADB_FixFetchedValues(MADB_Stmt *Stmt, int RowNumber, MYSQL_ROWS *Save
 SQLRETURN MADB_StmtFetch(MADB_Stmt *Stmt, my_bool KeepPosition)
 {
   unsigned int j, rc;
-  SQLULEN      ArraySize=  Stmt->Ard->Header.ArraySize;
-  MADB_Desc   *ArdDesc=    Stmt->Ard;
+  SQLULEN      ArraySize;
+  MADB_Desc   *ArdDesc;
   MYSQL_ROWS  *SaveCursor= NULL;
     
   if (!Stmt || !Stmt->stmt)
```

Comment:
Same here - !Stmt is safe to remove from here, and leave only !Stmt->stmt, and leave initialization as is